lib/ansible/modules/packaging/os/yum.py

```diff
@@ -253,6 +252,11 @@
   yum:
     name: sos
     disablerepo: "epel,ol7_latest"
+
+- name: Install a list of packages
+  yum:
+    name: "nginx,postgresql,postgresql-server"
+    state: latest
```

Comment:
Passing a comma separated list of packages works, but it's not something that we should promote. It's better to promote that `name` allows passing a list. For example:
```yml
- name: Install a list of packages
  yum:
    name:
      - nginx
      - postgresql
      - postgresql-server
    state: present
```
Also you should use the `present` state, because `latest` will update these packages, not only install them.